docker-compose.yml

```diff
@@ -17,6 +17,7 @@ services:
       DEBUG: "true"
       AIOHTTP_NOSENDFILE: "true"
       IS_LOCAL: "true"
+      COOKIE_KEY: 02a6b302c4290605f439baba456f954d
```

Comment:
Not needed. But im ok with it. It makes things better if you do a lot of restarts.